com.zoho.Notebook.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: com.zoho.Notebook
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+command: notebook
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+rename-desktop-file: notebook.desktop
+rename-icon: notebook
+modules:
+  - shared-modules/libsecret/libsecret.json
+
+  - name: notebook
+    buildsystem: simple
+    build-commands:
+      - ar -x Notebook*.deb
+      - rm -f Notebook*.deb
+      - tar -xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r opt/* usr/* $FLATPAK_DEST
+      - rm -rf opt usr
+      - install notebook.sh $FLATPAK_DEST/bin/notebook
+      - install -D -m 644 -t $FLATPAK_DEST/share/metainfo $FLATPAK_ID.metainfo.xml
+      - desktop-file-edit --set-key=Exec --set-value="notebook %U" $FLATPAK_DEST/share/applications/notebook.desktop
+    sources:
+      - type: file
+        url: https://downloads.zohocdn.com/notebooklinux-desktop/Notebook-2.0.4.deb
```

Comment:
are you sure this is architecture independent? because it is contains an x86_64 binary, this will be broken on aarch64.

Reply:
Actually, I don't. At first it had the arches rules but I removed because I wasn't sure. I will just add it again, since I don't have any way to test in another arch.